src/navigation/nav2_simple_navigator/src/simple_navigator.cpp

```diff
@@ -59,9 +59,16 @@ SimpleNavigator::execute(const nav2_tasks::NavigateToPoseCommand::SharedPtr comm
   // localization, while the goal pose is from the incoming command
   auto endpoints = std::make_shared<nav2_tasks::ComputePathToPoseCommand>();
 
-  endpoints->start = robot_.getCurrentPose()->pose.pose;
-  endpoints->goal = command->pose;
-  endpoints->tolerance = 2.0;
+  geometry_msgs::msg::PoseWithCovarianceStamped::SharedPtr current_pose;
+  if (robot_.getCurrentPose(current_pose)) {
+    endpoints->start = current_pose->pose.pose;
+    endpoints->goal = command->pose;
+    endpoints->tolerance = 2.0;
```

Comment:
What is the tolerance of 2.0 here? This should be a parameter.

Reply:
@mjeronimo This is outside the scope of robot class.  We should open a separate issue for this.

Reply:
Opened #164 to address this.